SCClassLibrary/Common/Control/Bus.sc

```diff
@@ -69,11 +70,14 @@ Bus {
 				["/c_setn", index, values.size] ++ values);
 		}, { error("Cannot set an audio rate bus") });
```

Comment:
and while you are at it, you could remove the trailing semicolons: `}, { error("Cannot set an audio rate bus") })`